slides/intro-02-data-budget.qmd

```diff
@@ -276,89 +263,103 @@ Make a plot or summary and then share with neighbor
 
 ## 
 
-```{r taxi-tip-counts}
+```{r forested-forested-counts}
 #| fig-align: 'center'
-taxi_train %>% 
-  ggplot(aes(x = tip)) +
+forested_train %>% 
+  ggplot(aes(x = forested)) +
   geom_bar()
 ```
 
 ## 
 
-```{r taxi-tip-by-local}
+```{r forested-by-tree-no-tree}
 #| fig-align: 'center'
-taxi_train %>% 
-  ggplot(aes(x = tip, fill = local)) +
+forested_train %>% 
+  ggplot(aes(x = forested, fill = tree_no_tree)) +
   geom_bar() +
   scale_fill_viridis_d(end = .5)
 ```
 
 ## 
 
-```{r taxi-tip-by-hour}
+```{r forested-by-precip-annual}
 #| fig-align: 'center'
-taxi_train %>% 
-  ggplot(aes(x = hour, fill = tip)) +
-  geom_bar()
+#| message: false
+#| warning: false
+forested_train %>% 
+  ggplot(aes(x = precip_annual, fill = forested, group = forested)) +
+  geom_histogram() +
+  scale_fill_viridis_d(end = .5)
 ```
 
+<!-- TODO: should we `position = "identity", alpha = .5` here? -->
+
 ## 
 
-```{r taxi-tip-by-hour-fill}
+```{r forested-by-precip-annual-fill}
 #| fig-align: 'center'
-taxi_train %>% 
-  ggplot(aes(x = hour, fill = tip)) +
-  geom_bar(position = "fill")
+#| message: false
+#| warning: false
+forested_train %>% 
+  ggplot(aes(x = precip_annual, fill = forested, group = forested)) +
+  geom_histogram(position = "fill") +
+  scale_fill_viridis_d(end = .5)
 ```
 
 ## 
 
-```{r taxi-tip-by-distance}
+```{r forested-forested-by-lat-lon}
 #| fig-align: 'center'
-taxi_train %>% 
-  ggplot(aes(x = distance)) +
-  geom_histogram(bins = 100) +
-  facet_grid(vars(tip))
+forested_train %>% 
+  ggplot(aes(x = lon, y = lat, col = forested)) +
+  geom_point() + 
+  scale_color_viridis_d(end = .5)
 ```
 
+TODO: should we lean into a `forested` color scheme and have green for forested, tan for not?
+
 # Split smarter
 
 ##
 
-```{r taxi-tip-pct, echo = FALSE}
-taxi %>%
-  ggplot(aes(x = "", fill = tip)) +
+```{r forested-forested-pct, echo = FALSE}
+forested %>%
+  ggplot(aes(x = "", fill = forested)) +
   geom_bar(position = "fill") +
-  labs(x = "")
+  labs(x = "") + 
+  scale_fill_viridis_d(end = .5)
 ```
 
 Stratified sampling would split within response values
 
 :::notes
-Based on our EDA, we know that the source data contains fewer `"no"` tip values than `"yes"`. We want to make sure we allot equal proportions of those responses so that both the training and testing data have enough of each to give accurate estimates.
+Based on our EDA, we know that the source data contains fewer `"No"` forested values than `"Yes"`. We want to make sure we allot equal proportions of those responses so that both the training and testing data have enough of each to give accurate estimates.
+
+TODO: this class imbalance is much less present with forested vs taxi -- should we keep it around?
```

Comment:
New TODO in discussion of accuracy:

> this class imbalance is much less present with forested vs taxi -- should we keep it around?

`forested` is about 55/45 now instead of 85/15ish for `tip`.

Reply:
I lean towards removing it here. The utility for this dataset isn't really there and then it risks becoming noise in this context even though it's generally useful to know